tools/pharmcat/pharmcat.xml

```diff
@@ -1,13 +1,18 @@
 <tool id="pharmcat" name="pharmCAT" version="@WRAPPER_VERSION@+@VERSION_SUFFIX@" profile="20.01">
     <description>
-        Pharmacogenomics Clinical Annotation Tool
+        Pharmacogenomics Clinical Annotation Tool new and improved
```

Comment:
I would remove the "new and improved" the user will get the latest version by default and if they choose an old version will be notified that a new version exists.